.env.development

```diff
@@ -1,2 +1 @@
 REDIS_SIDEKIQ_URL=redis://redis:6379/0
-REDIS_CABLE_URL=redis://redis:6379/1
```

Comment:
This is probably fine. Are we officially taking a stand against ActiveCable (and therefore Websockets)? Or is this change addressed elsewhere with different tooling?

Reply:
We never referenced this environment variable so I didn't want it available unless we actively configure ActiveCable